src/Helpers/LocalMachineHelper.php

```diff
@@ -14,36 +15,43 @@
 class LocalMachineHelper
 {
     /**
-     * Execute the given command on the local machine and return the last line of output
+     * @var integer The number of seconds to wait on a command until it times out
+     */
+    const TIMEOUT = 3600;
+
+    /**
+     * Executes the given command on the local machine and return the exit code and output.
      *
      * @param string $cmd The command to execute
-     * @return string The last line of the output of the executed command.
+     * @return array The command output and exit_code
      */
     public function exec($cmd)
     {
-        return exec($cmd);
+        $process = $this->getProcess($cmd);
+        $process->run();
+        return ['output' => $process->getOutput(), 'exit_code' => $process->getExitCode(),];
     }
 
     /**
-     * Execute the given command on the local machine and return the exit code and raw output
+     * Executes a buffered command.
      *
      * @param string $cmd The command to execute
-     * @return array The command output and exit_code.
+     * @param callable $callback A function to run while waiting for the process to complete
+     * @return array The command output and exit_code
      */
-    public function execRaw($cmd)
+    public function execBuffered($cmd, $callback)
```

Comment:
Buffered implies that the method is not interactive. It is true that the output of this function is stored and returned to the caller, but that is different than the typical use of the word. I think it would be more clear to call this `execInteractive`, and provide the recommended callback function from the caller in the docblock commend, e.g.:
```
               function ($type, $buffer) use ($output) {
                    $output->writeln($buffer);
                }
```
Otherwise, it is hard to tell simply from the documentation what the difference between this method and the `exec` method is supposed to be.